modules/support-providing-diagnostic-data-to-red-hat.adoc

```diff
@@ -96,4 +96,14 @@ If an existing `toolbox` pod is already running, the `toolbox` command outputs `
 ----
 # redhat-support-tool addattachment -c 01234567 /host/var/tmp/my-diagnostic-data.tar.gz <1>
 ----
++
+ifndef::openshift-rosa,openshift-dedicated[]
+[NOTE]
+====
+The `redhat-support-tool` has been deprecated in RHEL 8 and will not be shipped in RHEL 9 onwards.
+See link:https://access.redhat.com/articles/445443[Red Hat Access: Red Hat Support Tool] for more complete information on `redhat-support-tool`.
```

Comment:
```suggestion
The `redhat-support-tool` tool has been deprecated in {op-system-base-full} 8.0 and will be removed in {op-system-base} 9.0 and later.
For more information, see link:https://access.redhat.com/articles/445443[Red Hat Access: Red Hat Support Tool].
```

- Consider using attributes for RHEL. For future referrence, see https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc 
- Since you are using an article (`the`) consider mentioning what kind `redhat-support-tool` is, for example, "tool".
- Versions are described as  -- `8.0`, `9.0`, `7.0`, instead of `8`, `9`
- "For more information, see....." You can follow this format for links.